src/Domain/TagsTimeDependent.cpp

```diff
@@ -25,12 +25,17 @@ void InertialFromGridCoordinatesCompute<Dim>::function(
         tnsr::I<DataVector, Dim, Frame::Inertial>>>&
         grid_to_inertial_quantities) noexcept {
   if (not grid_to_inertial_quantities) {
+    // We use a const_cast to point the data into the existing allocation
+    // inside `source_coords` to avoid copying.
```

Comment:
To be totally clear, the reason this referencing trick isn't dangerous is because the output cache of the compute tags isn't modifyable, right?

Reply:
Oh, yes. The output of compute items in immutable (modulo more const casts :P )

Reply:
To be sure, I'm going to add checks for this and also do a refactor of a test that will basically remove the `test_time_independent` part of the test and template the other function. I'll push a fixup for this :)